### DISCUSSION TITLE
Source of Gama 1.9.1 or below with JDK for macos

### DISCUSSION THREAD

**Discussion by MaminiainaIndrafo** [gama-platform/gama#239] [General]
Hello, does someone have .dmg of gama 1.9 or 1.8 below for macOS because i have project how run on this or it's not available on gama website, if they are, you can send me on my gmail maminiainaindrafo@gmail.com

Thanks

**Comment by hqnghi88** [comment]
Hi , you can find some others version here https://github.com/gama-platform/gama.resources/releases
